.gitignore

```diff
@@ -1,11 +1,5 @@
 *.iml
 .gradle
-/local.properties
-/.idea/caches
-/.idea/libraries
-/.idea/modules.xml
-/.idea/workspace.xml
-/.idea/navEditor.xml
-/.idea/assetWizardSettings.xml
+.idea
```

Comment:
Don't agree here. I prefer the previous configuration

Reply:
b150a24e2071a9d1635152ddf349f91069c4d374

Reply:
![image](https://user-images.githubusercontent.com/10363352/156161546-42bd696a-ef42-47ac-9e1f-2f674ffe432c.png)

These files are helpless, will be easily miss-committing.